csrc/scheduler/utils.h

```diff
@@ -646,5 +647,17 @@ int64_t getPersistentBufferSizeOfTensor(
     SchedulerRuntimeInfo& runtime_info,
     const PersistentBufferInfo& persistent_buffer_info);
 
+//! Calculates the shared memory overhead per block, including CUDA driver
```

Comment:
I don't understand what this comment means. Could you please define what the shared memory overhead is?

Reply:
Did you add anything?

Still the overall comment is unclear to me. 

> Defaults to using maximum threads per block (threads_per_block = -1),

What does it mean? 

> The callers can pass other values if they are sure about the max values used at the runtime. 

What values are you talking about and why?

Reply:
this change was also not updated to remote.
Refering to `reduction workspace which depends on the number of threads per block specified by the parameter threads_per_block`
revised:
```
//! The required shared memory size for a block inclues two parts: (1) smem
//! for persistent buffers and (2) overhead. The overhead includes space
//! reserved by the CUDA driver and reduction workspace which depends on the
//! number of threads per block specified by the parameter threads_per_block.
//! By default, the function uses the maximum allowed number of threads per
//! block (threads_per_block = -1) to calculate the overhead. The caller can
//! specify a different value if they are sure about the max value used at
//! runtime.
```